templates/engage/shared/_es_engage_form.html

```diff
@@ -19,12 +19,12 @@
           aria-label="E-mail de trabajo" />
       </li>
       <li>
-        <label for="company">Nombre de la compañía:</label>
+        <label class="is-required" for="company">Nombre de la compañía:</label>
         <input required id="company" name="company" maxlength="255" type="text"
           aria-label="Nombre de la compañía" />
       </li>
       <li>
-        <label for="title">Título del puesto de trabajo:</label>
+        <label class="is-required" for="jobTitle">Título del puesto de trabajo:</label>
```

Comment:
```suggestion
        <label class="is-required" for="jobTitle">Título del puesto de trabajo:</label>
```
Is there a reason some of these are being changed to `jobTitle`, instead of the other way around? It seems they are still being submitted as `title`

Reply:
The "for" field only tells the label which field it points to which should match the "id" of the input itself. The attribute that gets submitted is "name" attribute on input field.

Reply:
That is correct, and why I raised it. It no longer matches the 'id' of the `input`

Reply:
@petesfrench Good catch! I've updated the ids. As for why change them it to jobTitle and not the other way around is because at all the other places, its the other way round. To be consistent, one of the two had to be changed and this one was smaller of the two.